runtime-config-linux.md

```diff
@@ -1,3 +1,5 @@
+# Linux Runtime Configuration
```

Comment:
We should echo whatever we use in the TOC here.  With your current 159a3eac9fc, that's “Linux Specific Runtime Configuration” (although see [my suggested alteration][1]).

[1]: https://github.com/opencontainers/specs/pull/176/files#r39865436